plugin-server/src/cdp/templates/_transformations/bot-detection/bot-detection.template.test.ts

```diff
@@ -84,112 +73,130 @@ describe('bot-detection.template', () => {
             },
         })
 
-        const response = await tester.invoke(
-            {
-                userAgent: '$raw_user_agent',
-                customBotPatterns: '',
-            },
-            mockGlobals
-        )
+        const response = await tester.invoke(DEFAULT_INPUTS, mockGlobals)
 
-        expect(response.finished).toBe(true)
-        expect(response.error).toBeUndefined()
-        expect(response.execResult).toBeNull()
+        expect(response.finished).toBeTruthy()
+        expect(response.error).toBeFalsy()
+        expect(response.execResult).toBeFalsy()
     })
 
-    it('should not have false positives with similar names', async () => {
-        const legitUserAgents = [
-            'Mozilla/5.0 CustomRobot/1.0', // Not in our bot list
-            'BrowserBot Pro/2.0', // Not in our bot list
-            'SpiderMonkey/91.0', // JavaScript engine, not a crawler
-        ]
-
-        for (const userAgent of legitUserAgents) {
-            mockGlobals = tester.createGlobals({
-                event: {
-                    properties: {
-                        $raw_user_agent: userAgent,
-                    },
+    it('should detect bot in case-insensitive manner', async () => {
+        mockGlobals = tester.createGlobals({
+            event: {
+                properties: {
+                    $raw_user_agent: 'Some-CRAWLER-Agent/1.0',
                 },
-            })
+            },
+        })
 
-            const response = await tester.invoke(
-                {
-                    userAgent: '$raw_user_agent',
-                    customBotPatterns: '',
-                },
-                mockGlobals
-            )
+        const response = await tester.invoke(DEFAULT_INPUTS, mockGlobals)
 
-            expect(response.finished).toBe(true)
-            expect(response.error).toBeUndefined()
-            expect(response.execResult).toBeDefined()
```

Comment:
This should have been failing, `response.execResult` was `null`, let's not use `toBeDefined` in general as it's easy to make this mistake.

This test *should* have been failing because `bot` is one of the patterns that is blocked by the template.